topics/proteomics/tutorials/metaproteomics/tutorial.md

```diff
@@ -126,8 +127,6 @@ The created dataset collection of the three *MGF files* in the history is used a
 >    - **X!Tandem: Protein stP Bias**: `No`
 >    - **X!Tandem: Maximum Valid Expectation Value**: `100`
 >
->    - leave everything else as default
->
 > 2. Click **Execute**.
 >
 {: .hands_on}
```

Comment:
In the peptideshaker options you need to change: 

Specify Advanced Filtering Options -> Advanced Filtering Options
Maximum Precursor Error Type -> Daltons

Otherwise Peptideshaker gives an error

Reply:
I cannot reproduce this - it doesn't even change the output at all for this data. It would make sense to select it here as well, but since it didn't change the output I expected it to figure that out with the default settings. Can you provide me with a failing history and/or the tool versions you used?

Reply:
Added this option, thanks!